internal/db/persister.go

```diff
@@ -14,6 +16,11 @@ func NewPersister(config Config) (persistence.Persister, error) {
 		err       error
 	)
 	switch config.Dialect {
+	case DialectMariaDB:
+		// See mysql.MySQL on why MariaDB is not supported.
+		err = errors.New("MariaDB is currently unsupported")
```

Comment:
Since we're using this string 4 times in this package, what do you think about creating an error and reuse it?

Reply:
Yeah, hopefully, it won't last long, but went ahead and added it in [e00582d](https://github.com/Kong/koko/pull/455/commits/e00582d58f8f394a90387480c6631084ea8c450a).